edgelet/edgelet-core/src/settings.rs

```diff
@@ -459,6 +464,59 @@ impl Listen {
     pub fn management_uri(&self) -> &Url {
         &self.management_uri
     }
+
+    pub fn min_tls_version(&self) -> Option<&Protocol> {
+        self.min_tls_version.as_ref()
+    }
+}
+
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Also `derive(Copy)`

Reply:
Why do we need Copy? Is it some kind of guideline?

Reply:
It's good to have types that can be Copy be Copy. Then clippy can enforce you pass them to functions by value instead of by pointer.

Reply:
Found this one https://stackoverflow.com/questions/44695216/should-simple-enums-derive-copy-is-there-any-difference-in-performance